.circleci/scripts/git-diff-develop.js

```diff
@@ -0,0 +1,76 @@
+const { promisify } = require('util');
+const exec = promisify(require('child_process').exec);
+const fs = require('fs');
+const path = require('path');
+
+
+async function fetchWithDepthIncrement(depth) {
```

Comment:
Nit: It would be create if these had JSDoc comments

Reply:
documentation added :+1: